app/namespace.go

```diff
@@ -683,15 +681,10 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				},
 				&cli.StringFlag{
 					Name:     namespaceRegionFlagName,
-					Usage:    "New region to add to the namespace.",
+					Usage:    "New region to add to the namespace. Please use format <cloud provider>-<region name>.",
```

Comment:
```suggestion
					Usage:    "New region to add to the namespace. Please use format <cloud provider>:<region name>.",
```

Reply:
This is the new cloud api format. I cannot change it.